tests/particle/test_particle.py

```diff
@@ -43,11 +43,15 @@ def test_lambda_style_search():
     assert 1000010010 in particles
     assert -1000010010 in particles
 
-    [p.pdgid for p in Particle.findall(lambda p: p.pdg_name == "p" and p > 0)] == [
+    assert [
+        p.pdgid for p in Particle.findall(lambda p: p.pdg_name == "p" and p > 0)
+    ] == [
```

Comment:
This originally did nothing, was a pointless comparison (and several following it).

Reply:
Ah, the `assert` was missing :S!